src/signals/incident-management/components/CheckboxList/index.js

```diff
@@ -109,10 +110,12 @@ const CheckboxList = ({
    * @returns {Boolean}
    */
   const isDefaultChecked = id => defaultValue.findIndex(value => value.id === id || value.key === id) >= 0;
+  // const checkForAllChecked = () => defaultValue.length === options.length;
+  const checkDefault = () => (groupId && isDefaultChecked(groupId)) || (!groupId && defaultValue.length === options.length);
```

Comment:
The name is confusing. I'd opt for `isGroupChecked` and make it a scalar value instead of a function.

Reply:
gedaan